src/main/java/me/totalfreedom/totalfreedommod/command/Command_wipecoreprotectdata.java

```diff
@@ -0,0 +1,31 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitRunnable;
+
+@CommandPermissions(level = Rank.TELNET_ADMIN, source = SourceType.ONLY_CONSOLE)

```

Comment:
Providing everything else is working as expected, I dont think we need the command, server owners and developers with privileged user access will have access to the DB its self, so could run SQL queries and statements to wipe select parts of the database, though I would like a second opinion from @JeromSar or @StevenLawson on this.